Assets/Scripts/Game/StageManager.cs

```diff
@@ -55,8 +52,25 @@ public IReactiveProperty<float> CurValue
     }
     
     private float timer = 0f;
-    
-    
+
+
+    private void Update()
+    {
+        if (GameManager.Instance.IsGame.Value == false)
+            return;
+
+        if (this.stageInfo.Type == Define.StageType.LimitTime)
+        {
+            timer += Time.deltaTime;
+            if (timer >= 0.1f)
+            {
+                // 1초에 1씩 깎는 것
+                this.curValue.Value -= 100f;
+                timer = 0f;
```

Comment:
```suggestion
                timer -= 0.1f;
```
이게 조금 더 정확합니다. 0.1초에서 넘어가는 값을 보존할 수 있습니다.